### PR TITLE
Bug/compound topography

### PR DESCRIPTION
**Pull request by sannant**

### CODE REVIEW

**Comment by sannant** [comment]
@pastewka , once this branch is merged we should do a release, since it is currently blocking ContactMechanics and the tip artefacts are a major issue

File: SurfaceTopography/ChangeLog.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Change log for SurfaceTopography
 v1.3.1 (not yet released)
```
Comment (pastewka) [inline SurfaceTopography/ChangeLog.md:4]: 1.3.1 was released Jan 14

**Review by pastewka — CHANGES_REQUESTED** [review]

File: SurfaceTopography/ChangeLog.md (hunk 1 above)
Comment (sannant) [reply SurfaceTopography/ChangeLog.md]: Ah sorry I forgot to pull

**Review by pastewka — APPROVED** [review]
Should I do a new release?

**Pull request merged**